flowgate-common/src/main/java/com/vmware/flowgate/common/FlowgateConstant.java

```diff
@@ -42,6 +42,14 @@
     */
    public static String NETWORK_PORT_FOR_SERVER = "NETWORK_PORT_FOR_SERVER";
 
+   public static String PDU_OUTLETS_FROM_POWERIQ = "PDU_OUTLETS_FROM_POWERIQ";
+
+   public static String PDU_RATE_POWER_FROM_POWERIQ = "PDU_RATE_POWER_FROM_POWERIQ";
```

Comment:
There is no need to specify PowerIQ, since pdu rate power is always consistent across different system.